CONTRIBUTING.md

```diff
@@ -131,6 +138,14 @@ Start Paralus:
 go run github.com/paralus/paralus
 ```
 
+#### Updating Proto Files?
+
+If you're making changes to any of the proto files, you'll have to perform the following steps:
+
+- `make clean` - would clean all the *.pb files
+- `make build-proto` - would ensure changes to proto are valid
```

Comment:
Making some changes to the Makefile in https://github.com/paralus/paralus/pull/32 . Will rework this once that is merged.

Reply:
- `build-proto`: rebuilds proto files
- `clean`: clean binary artifacts

No more gen proto